datacite-doi-commons/src/test/java/no/unit/nva/datacite/DynamoRecordDtoMapperTest.java

```diff
@@ -0,0 +1,22 @@
+package no.unit.nva.datacite;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+
+import no.unit.nva.publication.doi.dto.Publication;
+import no.unit.nva.publication.doi.dto.PublicationDtoTestDataGenerator;
+import no.unit.nva.transformer.dto.DynamoRecordDto;
+import org.hamcrest.Matchers;
+import org.junit.jupiter.api.Test;
+
+public class DynamoRecordDtoMapperTest {
```

Comment:
what happens if there is an empty field? Do we log or send an exception with some details?

Reply:
I think I map this as is, the transformer will uncover any missing values